src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -19,7 +20,7 @@ type BioAndDMSettingProps = {
   onlyReceivedDmFromUserFollowing: boolean;
 };
 
-const CheckBoxCustom = (props: {value: boolean; label: string}) => {
+const CheckBoxCustom = (props: {value: boolean; label: string; disabled?: boolean}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `disabled` prop to the `CheckBoxCustom` component is a good change. It allows for better control over the interactivity of the checkbox.



<!-- This is an auto-generated comment by OSS CodeRabbit -->